exporters/zipkin/src/zipkin_exporter.cc

```diff
@@ -98,6 +98,12 @@ void ZipkinExporter::InitializeLocalEndpoint()
   local_end_point_["port"] = url_parser_.port_;
 }
 
+bool ZipkinExporter::Shutdown(std::chrono::microseconds timeout) noexcept
+{
+  isShutdown_ = true;
```

Comment:
Not related to PR, but probably we need to discuss if this should be an atomic operation for all exporters to protect against Shutdown() and Export() called from separate threads. While specs don't allow Export() to be called concurrently ( https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/sdk.md#exportbatch ), it doesn't talk about Export() and Shutdown().

Reply:
Agree it should be protected somehow.